hyperspy/io_plugins/emd_fei.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2015 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import warnings
+
+import h5py
+import numpy as np
+from traits.api import Undefined
+
+from hyperspy.misc.utils import ensure_unicode
+from hyperspy.axes import AxesManager
+
+import json
+
+
+# Plugin characteristics
+# ----------------------
+format_name = 'EMD FEI'
+description = \
+    'File reader for the EMD FEI format'
+
+full_support = False
+# Recognised file extension
+file_extensions = ['emd']
+default_extension = 0
+
+# Writing capabilities
+writes = False
+
+def file_reader(filename, mode='r', driver='core',
+                backing_store=False, **kwds):
+    with h5py.File(filename, mode=mode, driver=driver) as f:
+        experiments = []
+        exp_dict_list = []
+        for experiment_key, experiment_group in f.iteritems():
+            if isinstance(experiment_group, h5py.Group):
+                exp_dict = emdimagegroup2signaldict(experiment_key, experiment_group)
+                exp_dict_list.extend(exp_dict)
+        return exp_dict_list
+
+def _get_emd_metadata(metadata_string):
+    original_metadata_dict = json.loads(metadata_string)
```

Comment:
I'll add a check for that later, and check if the number of axes dimensions in the metadata is equal to the data dimensions. If not, it will generate some "default" axes with scale=1, offset=1, units=''.